README.md

```diff
@@ -239,8 +193,20 @@ All output from the console is synchronized to the file in real-time.
 
 If you close the program and then reopen it, a new log file will be created.
 
+
 ## Run the test
 
 You could write tests for your combination of commands in the `test.py` file. Simply run `python3 test.py` on the console to perform the tests. When the test script is started, it will first **kill any existing `pvm.py` processes**, and then spawn a new one to test the commands, and end it again after test is finished.
 
-## Known limitations
+
+## Helpful links
+- https://python-omxplayer-wrapper.readthedocs.io/en/latest/
+
+Also don't forget to checkout our [wiki](https://github.com/omarcostahamido/PVM/wiki)! It contains instructions on various topics like [_setting up remote access to the Raspberry Pi_](https://github.com/omarcostahamido/PVM/wiki/How-to-connect-to-a-Raspberry-Pi-remotely) ,[_VS Code ssh to Raspberry Pi_](https://github.com/omarcostahamido/PVM/wiki/VS-Code-ssh-to-Raspberry-Pi) and [_The workflow in MAX_](https://github.com/omarcostahamido/PVM/wiki/The-workflow-in-MAX).
```

Comment:
Though this list here is nice. I would still say that some of the wiki pages would be much more helpful to link (also) when it can be needed the most. I.e. somewhere in the installation process just mention that `for more information about...` or `see here{hyperlink} how to do...`

Reply:
We keep this now, and I will add those wiki links to where it is needed most.